cms/templates/activation_complete.html

```diff
@@ -9,8 +9,8 @@
 <div class="wrapper-mast wrapper sr">
   <header class="mast">
     <h1 class="page-header">
-      ${_("{studio_name} Account Activation").format(
-        studio_name=Text(settings.STUDIO_SHORT_NAME)
+      ${Text(_("{studio_name} Account Activation")).format(
+        studio_name=Text(settings.STUDIO_SHORT_NAME),
```

Comment:
@robrap @dan-f Sorry, I am still a bit confused here. After the separate offline conversations I had with each of you, I came away understanding two things with respect to the specific case commented on here (String formatting and interpolation with no HTML involved):

1. Adding the `Text` function has no effect.
2. The best practice in this case, when there is no HTML involved, is to NOT add the Text function.

Please let me know if I have the wrong understanding.